ci-operator/step-registry/openstack/ovsdpdk/machineset/openstack-ovsdpdk-machineset-commands.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
```

Comment:
you'll need to load the clouds.yaml here:

```
export OS_CLOUD
export OS_CLIENT_CONFIG_FILE="${SHARED_DIR}/clouds.yaml"
```

Reply:
instead we'll just take the network id as input to the step. Either directly or from shared_dir. We have to create the network anywahys. and using the name to look it up is a wasted cycle.